scripts/install.sh

```diff
@@ -25,3 +25,12 @@ if [[ "${TOX_ENV}" == "pypy" ]]; then
     pyenv install pypy-2.6.0
     pyenv global pypy-2.6.0
 fi
+
+if [[ "${TOX_ENV}" == "gae" && ! -d ${GAE_PYTHONPATH} ]]; then
+    mkdir -p `dirname ${GAE_PYTHONPATH}`
+    cd `dirname ${GAE_PYTHONPATH}`
+    wget https://storage.googleapis.com/appengine-sdks/featured/google_appengine_1.9.25.zip
+    unzip -q google_appengine_1.9.25.zip
+    # Directory is required for the write to well known file test to pass.
+    mkdir -p $HOME/.config/gcloud
```

Comment:
Can we leave that for another CL? We can mock os.exists or whatever in that test.